downstairs/src/region.rs

```diff
@@ -109,6 +109,57 @@ impl Inner {
             .execute("UPDATE metadata SET value=1 WHERE name='dirty'", [])?;
         Ok(())
     }
+
+    fn get_encryption_context(
```

Comment:
If/when we turn this on for real, we should add to the `dump` command something that will print this out.  I don't know exactly how that would look yet, but I know when building and debugging the reconciliation, we may want to look at and compare the nonce/tags between different downstairs.

Reply:
I missed adding this and did so in 27b2365. `dump` now looks at nonces and tags, and also has a "DIFF" column to highlight anything it found. Here's an example:

```
$ ./dump.sh -d disks/d1/ -d disks/d2/ -d disks/d3/ -e 0 | head -n20
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
     Running `target/debug/crucible-downstairs dump -d disks/d1/ -d disks/d2/ -d disks/d3/ -e 0`
           Extent 0
GEN             0        0        0 
FLUSH_ID       78       78       78 
DIRTY           D        D        D 

             Data 0      Data 1      Data 2      Nonce 0     Nonce 1     Nonce 2      Tag 0       Tag 1       Tag 2     DIFF  
Block    0     A           A           A           A           A           A           A           A           A             
Block    1     A           A           A           A           B           B           A           B           B      <-------
Block    2     A           A           A           A           A           A           A           A           A             
Block    3     A           A           A           A           A           A           A           A           A             
Block    4     A           A           A           A           A           A           A           A           A             
Block    5     A           A           A           A           A           A           A           A           A             
Block    6     A           A           A           A           A           A           A           A           A             
Block    7     A           A           A           A           A           A           A           A           A             
Block    8     A           A           A           A           A           A           A           A           A             
Block    9     A           A           A           A           A           A           A           A           A             
Block   10     A           A           A           A           A           A           A           A           A             
Block   11     A           A           A           A           A           A           A           A           A             
Block   12     A           A           A           A           A           A           A           A           A             
Block   13     A           A           A           A           A           A           A           A           A
```